src/models/VerificationResult.ts

```diff
@@ -0,0 +1,7 @@
+interface VerificationResult {
+  nonce?: string;
```

Comment:
are these fields optional?

Reply:
No it's not.

Reply:
I mean the fields, not the object. Is `nonce` or `token` optional or not?

Reply:
no it's not